zerver/tests/test_bugdown.py

```diff
@@ -444,7 +444,7 @@ def test_realm_patterns(self):
                                    url_format_string=url_format_string)
         realm_filter.save()
         self.assertEqual(
-            str(realm_filter),
+            realm_filter.__unicode__(),
```

Comment:
`str(realm_filter)` - this doesn't work with python 3
We need to use `python_2_unicode_compatible` decorator, but it requires
`from __future__ import unicode_literals`
and this will require many changes in models.py, for this reason I added this hack